Jenkinsfile_CNP

```diff
@@ -1,20 +1,28 @@
 #!groovy
 
+properties([
+  // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
+  pipelineTriggers([cron('H 04 * * *')]),
+  parameters([
+    string(name: 'TEST_URL', defaultValue: 'https://nfdiv.aat.platform.hmcts.net', description: 'The URL you want to run tests against'),
```

Comment:
/?normally you would have this for nightly. Do you want Jenkins trigger cron for main pipeline ?

Reply:
I've swapped out the main pipeline with the nightly pipeline just for debugging at the moment, which is why I've marked this PR as draft 😜